src/dynamic-page.scss

```diff
@@ -0,0 +1,359 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-dynamic-page;
+
+.#{$block} {
+
+  $fd-dynamic-page-background-color: var(--sapObjectHeader_Background);
+  $fd-dynamic-page-content-background-color: var(--sapBackgroundColor);
+  $fd-dynamic-page-padding-top-bottom: 0.5rem;
+  $fd-dynamic-page-header-padding-top-bottom: 1rem;
+  $fd-dynamic-page-box-shadow: var(--sapContent_HeaderShadow);
+  $fd-dynamic-page-button-width: 1.5rem;
+  $fd-dynamic-page-button-padding: 0 0.25rem;
+  $fd-dynamic-page-button-group-width: 4rem;
+  $fd-dynamic-page-button-group-height: 0.0625rem;
+  $fd-dynamic-page-focus-outline-offset: 0.0625rem;
+
+  @include fd-reset();
+
+  @mixin responsive-page($padding-y: 0,$padding-x: 1rem) {
+    padding: $padding-y $padding-x;
+  }
+
+  @mixin no-shadow() {
+    box-shadow: none;
+  }
+
+  @mixin custom-button() {
+    @include fd-flex-vertical-center() {
+      justify-content: center;
+    }
+
+    margin: $fd-dynamic-page-button-padding;
+    padding: 0;
+    width: $fd-dynamic-page-button-width;
+    height: 2rem;
+    min-width: $fd-dynamic-page-button-width;
+
+    @include fd-rtl() {
+      > [class*='sap-icon'] {
+        &:last-child {
+          margin-right: 0;
+          margin-left: 0;
+        }
+      }
+    }
+  }
+
+  @include fd-flex(column);
+
+  &__title-area {
+    @include fd-reset();
+
+    @include fd-flex(column) {
+      justify-content: space-between;
+    }
+
+    @include fd-fiori-focus(0.0625rem);
+
+    min-height: 3rem;
+    word-wrap: break-word;
+    background-color: $fd-dynamic-page-background-color;
+    z-index: 1;
+
+    @include fd-hover() {
+      background: var(--sapObjectHeader_Hover_Background);
+      border-bottom: var(--fdDynamicPage_Hover_Border_Bottom);
+      cursor: pointer;
+    }
+
+    &--s {
```

Comment:
we agreed to use `sm` for small, `md` for medium, `lg` for large. Can you refactor the class names and documentation?

Reply:
done